### ISSUE TITLE
Confusing output if project default is used the specify default lib config

### ISSUE THREAD

**Issue by Nicoretti** [esrlabs/bake#75]
If the default keyword is used to specify a default "Build" target for a project in the Project.meta
like:
```
Project default: someExecutableConfig {
...
}
```
the target specified with default will be build => expected behaviour.

But if the default keyword is used to specify the default library config which will be selected by dependend project(s) it gets confusing.

If the bake command is invoked without any arguments for such a case:
```
Project default: Lib {

    Responsible {
        Person "Nicola Coretti", email: "nicola.coretti@esrlabs.com"
    }

    LibraryConfig IncludeOnly {
        IncludeDir "include", inherit: true
    }

    LibraryConfig Lib {
        Files "src/**/*.cpp"
        IncludeDir "include", inherit: true
...
}
```
an error is reported:
```
NiCoretti@Nc-ESRLabs: ~/VM_Projects/E.S.R.Labs/bsw/util
◖Tue 8 | 11:46◗ >>> bake
-- bake 2.16.1, ruby 2.1.2p95, platform x86_64-darwin13.0 --
Info: cache not found, reloading meta information
Loading /Users/NiCoretti/VM_Projects/E.S.R.Labs/bsw/util/Project.meta
/Users/NiCoretti/VM_Projects/E.S.R.Labs/bsw/util/Project.meta:11: Error: Main project configuration must contain DefaultToolchain

Building failed.
```

I would expected it to print the target list, due to the fact that it makes no sense to build a Lib by its own (without Toolchain!).

I think the main issue here is that the default keyword does have 2 different "meanings" depending on the context, therefore it should behave contextual sensitive in all cases or an unambiguous other keyword should be used.

**Comment by aschaal** [comment]
It is 1 keyword for 1 meaning, but used from 2 different places. Commandline and in Project.meta. So this is fine.
However, I should list the possible build targets in case the main config has to DefaultToolchain. I'll change that.

**Comment by aschaal** [comment]
Fixed with 2.17.1.
